lib/test/CDataFrameAnalysisSpecificationFactory.cc

```diff
@@ -374,7 +384,7 @@ CDataFrameAnalysisSpecificationFactory::predictionSpec(const std::string& analys
         m_CategoricalFieldNames, true, CTestTmpDir::tmpDir(), "ml", analysis,
         this->predictionParams(analysis, dependentVariable))};
 
-    LOG_TRACE(<< "spec =\n" << spec);
+    LOG_DEBUG(<< "spec =\n" << spec);
```

Comment:
Can we change this back to TRACE before committing: it generates quite a lot of noise in the test logs
```suggestion
    LOG_TRACE(<< "spec =\n" << spec);
```

Reply:
Oh definitely, I had that when I was debugging my tests :)